### PR TITLE
update docs links to mongodb docs

### PR DESCRIPTION
**Pull request by mhor**

### CODE REVIEW

File: docs/en/reference/find-and-update.rst [hunk 1]
```diff
@@ -74,7 +74,7 @@ You can also remove a document and return it:
         ->execute();
 
 You can read more about the find and modify functionality on the
-`MongoDB website <http://www.mongodb.org/display/DOCS/findandmodify+Command>`_.
+`MongoDB website <https://docs.mongodb.com/v3.0/reference/method/db.collection.findAndModify/>`_.
```
Comment (malarzm) [inline docs/en/reference/find-and-update.rst]: There's no need to pinpoint docs to 3.0, https://docs.mongodb.com/manual/reference/method/db.collection.findAndModify/ link can be used instead

**Comment by malarzm** [comment]
@mhor thanks a bunch for going through all the links! There's one change I'd like to see before merging though (please see comment above), would you mind ammending your commit with it?

File: docs/en/reference/find-and-update.rst (hunk 1 above)
Comment (mhor) [inline docs/en/reference/find-and-update.rst]: @malarzm Done :+1:

**Comment by malarzm** [comment]
Thank you @mhor!

**Pull request merged**